postgre-client/src/asset_filter_client.rs

```diff
@@ -303,7 +303,8 @@ fn add_filter_clause<'a>(
     }
 
     if !options.show_unverified_collections {
-        query_builder.push(" AND assets_v3.ast_is_collection_verified = true");
+        // if there is no collection for asset it doesn't mean that it's unverified
```

Comment:
But it also doesn't mean that it's verified :upside_down_face:
But as I understood other providers have such logic, so there just thoughts out loud

Reply:
The point in that change was that if there is no collection for an asset we cannot filter it by `collection_verified` field because there is no such field for asset at all.